### PR TITLE
Move cli exec flags to own file

### LINKED ISSUE #375: Move cli flags to separate file

### PR DESCRIPTION
**Pull request by 6543**
close  #375

### CODE REVIEW

File: cli/build/build_logs.go [hunk 1]
```diff
@@ -14,5 +14,5 @@ var buildLogsCmd = cli.Command{
 }
 
 func buildLogs(c *cli.Context) error {
-	return fmt.Errorf("Command temporarily disabled. See https://github.com/drone/drone/issues/2005")
+	return fmt.Errorf("Command temporarily disabled. See https://web.archive.org/web/20201023145707/https://github.com/drone/drone/issues/2005")
```
Comment (anbraten) [inline cli/build/build_logs.go:17]:
```suggestion
	// TODO: add logs command
	return fmt.Errorf("Command temporarily disabled. See https://github.com/woodpecker-ci/woodpecker/issues/383")
```

File: cli/exec/flags.go [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+			"plugins/docker",
+			"plugins/gcr",
+			"plugins/ecr",
+		},
+	},
+
+	//
+	// Please note the below flags are mirrored in the pipec and
+	// should be kept synchronized. Do not edit directly
+	// https://github.com/cncd/pipeline/pipec
```
Comment (anbraten) [inline cli/exec/flags.go:65]: I think we removed `pipec`? Should we just remove that command?

Comment (6543) [reply cli/exec/flags.go:65]: tbh no idear :)

Comment (6543) [reply cli/exec/flags.go:65]: tbh no idear :)

File: cli/exec/flags.go [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+		Value: &cli.StringSlice{
+			"plugins/docker",
+			"plugins/gcr",
+			"plugins/ecr",
+		},
+	},
+
+	//
+	// Please note the below flags are mirrored in the pipec and
+	// should be kept synchronized. Do not edit directly
```
Comment (anbraten) [inline cli/exec/flags.go:64]:
```suggestion
	// Please note the below flags should match the flags from
	// pipeline/frontend/metadata.go and should be kept synchronized.
```

File: cli/exec/flags.go (hunk 2 above)
Comment (anbraten) [reply cli/exec/flags.go:65]:
```suggestion
```

File: cli/exec/flags.go [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+		EnvVar: "WOODPECKER_PREV_COMMIT_AUTHOR_EMAIL",
+		Name:   "prev-commit-author-email",
+	},
+	cli.IntFlag{
+		EnvVar: "WOODPECKER_JOB_NUMBER",
+		Name:   "job-number",
+	},
+	cli.StringSliceFlag{
+		EnvVar: "WOODPECKER_ENV",
+		Name:   "env, e",
```
Comment (anbraten) [inline cli/exec/flags.go:264]: I checked the code: Only `env` is used.

```suggestion
		Name:   "env",
```

**Review by anbraten — APPROVED** [review]

**Pull request merged**